lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -198,7 +202,8 @@ private boolean hasMatchingGpg45Profile(
                                 vcs,
                                 ipAddress,
                                 deviceInformation));
-                ipvSessionItem.setVot(Vot.P2);
+
+                ipvSessionItem.setVot(Vot.fromGpg45Profile(matchedProfile.get()));
```

Comment:
This isn't your code, so just a comment - this is a big side effect to be hiding in this method

Reply:
I wrote the `fromGpg45Profile` method if that's what you mean? Or are you saying the ipv session VoT has a large side effect

Reply:
I mean that silently setting an important session property in method whose name reads like a simple test is ringing alarm bells.